tests/unit_tests/test_contract/contracts/contract_for_data_conversion_test.tz

```diff
@@ -0,0 +1,507 @@
+{ parameter
```

Comment:
When I was using the command-line version of Ligo (0.22.0) from <https://ligolang.org/docs/intro/installation/>, the source + storage combination was working fine, so seemed like good contenders here.